keywording/text.xml

```diff
@@ -94,7 +94,8 @@ Do <b>not</b> use the <c>*</c> or <c>~*</c> special keywords in ebuilds.
 <note>
 Usually, "live" ebuilds
 (see <uri link="::ebuild-writing/functions/src_unpack/vcs-sources/"/>)
-do not specify a <c>KEYWORDS</c> variable, or assign the empty string to it.
+do not specify a <c>KEYWORDS</c> variable, or assign the empty string to it
+(which is not strictly necessary).
```

Comment:
What do we want to say here? Either we're fine with both variants, then the wording is fine as it is, and the parenthesis is just adding noise.

Or, if we want to recommend that live ebuilds have no KEYWORDS line, then we should just do this:

```suggestion
do not specify a <c>KEYWORDS</c> variable.
```

Reply:
I agree, let's recommend it's not done.

Reply:
Suggestion not applied, though?

Reply:
Whoops.